mllib/src/main/scala/org/apache/spark/mllib/pmml/export/PMMLModelExportFactory.scala

```diff
@@ -44,12 +44,12 @@ private[mllib] object PMMLModelExportFactory {
         new GeneralizedLinearPMMLModelExport(lasso, "lasso regression")
       case svm: SVMModel =>
         new BinaryClassificationPMMLModelExport(
-          svm, "linear SVM", RegressionNormalizationMethodType.NONE,
+          svm, "linear SVM", RegressionModel.NormalizationMethod.NONE,
```

Comment:
weird, maybe I missed something obvious, but why is it a RegressionModel normalization method for something that is a classification model

Reply:
I think it's because the two classifiers it supports are logistic regression and SVM. The former makes sense but not so much SVM, but, it looks like only PMML 4.3 established an export format for it: http://dmg.org/pmml/v4-3/SupportVectorMachine.html . But now we're on PMML 4.3 with this change. That's probably worth doing, though my real purpose in this change is to get Java 9+ working. That can be handled separately.

Reply:
`RegressionModel` is a top-level PMML model element for representing all "dot product"-type models; the real mining function type (regression vs classification) is specified using the `functionName` attribute. So, `RegressionModel@functionName="classification"` is how communicates that this model element is encoding a classifier-type function.

Based on my experience with implementing Apache Spark ML-to-PMML converters (https://github.com/jpmml/jpmml-sparkml), then many non-decision tree based Apache Spark ML model classes (eg. `NaiveBayesClassifier`, `LinearSVC`) are based on the "dot product" business logic, and therefore reducible to the `RegressionModel` element. In other words, there is no point in using more complex PMML model elements such as `NaiveBayesModel` or `SupportVectorMachineModel`, when the simplest `RegressionModel` element will be able to capture everything.

Reply:
Thanks as always @vruusmann for review! glad we're getting on a newer version of JPMML